packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -54,17 +58,63 @@ class CupertinoScrollbar extends StatefulWidget {
   /// typically a [Scrollable] widget.
   final Widget child;
 
+  /// The [ScrollController] used to implement Scrollbar dragging.
+  ///
+  /// Starting in iOS13, a long press on the scroll bar or a drag in from the
```

Comment:
This paragraph and the next one are slightly misleading. We don't want developers that this feature will only be enabled on ios13; just trying to explain that the new behavior matches native ios13.